doc/source/ccs/building-a-case.rst

```diff
@@ -0,0 +1,135 @@
+.. _ccs_building_a_case:
+
+Building a Case
+===============
+
+.. contents::
+   :local:
+
+Once the case has been created and set up, it's time to build the executable.
+Several directories full of source code must be built all with the same compiler and flags.
+The **case.build** script performs all build operations (setting dependencies, invoking Make/CMake,
+creating the executable).
+
+Build Process
+-------------
+The ``case.build`` command generates the utility, component libraries and the model executable. Each utility 
+and component build has a log file generated named in the form: **$component.bldlog.$datestamp**. These are 
+located in ``$EXEROOT`` which can be retrieved using ``./xmlquery EXEROOT``. If the logs are compressed 
+(as indicated by a .gz file extension), the build ran successfully.
+
+Invoking ``case.build`` creates the following directory structure in ``$EXEROOT`` if the Intel compiler is used:
+
+::
+
+    atm/
+    cpl/
+    esp/
+    glc/
+    ice/
+    intel/
+    lib/
+    lnd/
+    ocn/
+    rof/
+    wav/
+
+Except for **intel/** and **lib/**, each directory contains an **obj/** subdirectory for the target model component's compiled object files.
+
+The *mct*, *pio*, *gptl*, and *csm_share* libraries are placed in a directory tree that reflects their dependencies. See the **bldlog** for a given component to locate the library.
```

Comment:
This might be a bit deceptive.   For example cesm will not have an mct directory and if pio is provided as a module there will be no pio directory either.  There may also be other optional directories here.

Reply:
@jedwards4b I realized there was a little redundancy in this page, I cleaned it up and made sure to note that this is just an example output and can vary depending on the model, compiler, etc.